src/snake_game/handlers.cljs

```diff
@@ -1,86 +0,0 @@
-(ns snake-game.handlers
-  (:require-macros [reagent.ratom :refer [reaction]])
-  (:require [re-frame.core :refer [register-handler
-                                   register-sub
-                                   dispatch]]
-            [goog.events :as events]
-            [snake-game.utils :as utils]))
-
-;; Define app data. We need to define our board, points and snake. In our snake vector is in map desctibed position :position of every 'snake body' part. First elemeny in vector id head of snake. Key :direction is the direction of the next move.
-
-(def board [35 25])
-
-(def snake {:direction [1 0]
-            :body [[3 2] [2 2] [1 2] [0 2]]})
-
-(def initial-state {:board board
-                    :snake snake
-                    :point (utils/rand-free-position snake board)
-                    :points 0
-                    :game true})
-
-(register-handler                  ;; setup initial state
- :initialize                       ;; usage (submit [:initialize])
- (fn
-   [db _]
-   (merge db initial-state)))      ;; what it returns becomes the new @db state
-
-(register-handler
- :next-state
- (fn
-   [db _]
-   (if (:game db)
-     (if (utils/collisions db)
-       (assoc-in db [:game] false)
-       (-> db
-           (update-in [:snake] utils/move-snake)
-           (as-> after-move
-               (utils/process-move after-move))))
-     db)))
-
-(register-handler
- :change-direction
- (fn [db [_ new-direction]]
-   (update-in db [:snake :direction] (partial utils/change-snake-direction
-                                             new-direction))))
-
-;;Register global event listener for keydown event.
-;;Processes key strokes according to `utils/key-code->move` mapping
-(defonce key-handler
-  (events/listen js/window "keydown"
-                 (fn [e]
-                   (let [key-code (.-keyCode e)]
-                     (when (contains? utils/key-code->move key-code)
-                       (dispatch [:change-direction (utils/key-code->move key-code)]))))))
-
-;; ---- Subscription Handlers ----
-
-(register-sub
```

Comment:
I see what you are doing here, but you might consider removing `:board` from the reactive states. At the end of the day, it is not changing at all in this implementation...your reader can ask themselves why you did it. Alternatively, you add a chooser for the board size (increasing the fun! :smile:)